### ISSUE TITLE
Performance difference in Diving48 dataset

### ISSUE THREAD

**Issue by Fazlik995** [swathikirans/GSM#13]
Hi, Swathikirans

Thank you for sharing your nice work.

I trained your algorithm using Diving48 dataset, however, my result is lower than 40.27%.

Below my configuration:
python3 main.py diving48 RGB --split 1 --arch InceptionV3 --num_segments 16 --consensus_type avg --batch-size 8 --iter_size 1 --dropout 0.7 --lr 0.01 --warmup 10 --epochs 20 --eval-freq 5 --gd 20 --run_iter 1 -j 16 --npb --gsm

Can not understand why I am receiving poor performance.

Any ideas? Please, if you can help me.

Thank you!

**Comment by swathikirans** [comment]
What is the accuracy you are getting? Can you try training for more epochs, say 30?

**Comment by Fazlik995** [comment]
Thank you for fast reply.

My best accuracy so far is 37.40%.

I received above accuracy after changing "args.epochs" to "args.epochs+1" in line 151.

I will try to train using more epochs as you suggested.

**Comment by swathikirans** [comment]
The result reported in the paper is obtained using 2 clips during inference. The accuracy you obtained should improve further using this inference strategy.

**Comment by Fazlik995** [comment]
Actually, I got 37.40% after testing 2 clips during inference.

**Comment by Fazlik995** [comment]
I compare my log file and your log file.

I noticed that my learning rate was the same as up to 10 epochs. After 10th epochs, my learning rate starts decreasing slowly than yours.

At 19 epoch you have lr: 0.00024, I have lr: 0.00079.

The problem occurs because of this learning rate issue I guess.

What do you think?

**Comment by swathikirans** [comment]
You mentioned earlier that you changed  "args.epochs" to "args.epochs+1" in line 151. That is why the learning rate is different than mine. The learning rate scheduler adjusts the learning rate depending on the number of epochs. So if the epochs are different, then the learning rates will also be different.

**Comment by Fazlik995** [comment]
I will train using more epochs and tell the result.

I have one more question regarding the something-something-v1 dataset problem.

I downloaded the dataset from the official website. When I downloaded the dataset my first file 20bn-something-something-v1-00 is gzip file format. However, from  20bn-something-something-v1-01 downloaded files are in binary format.

When I extract using cat 20bn-something-something-v1-01 | tar zx i got the following error:

gzip: stdin: not in gzip format
tar: Child died with signal 13
tar: Error is not recoverable: exiting now

Please, help me to solve this issue.
I could not solve this problem

Thank you.

**Comment by swathikirans** [comment]
Use the following command to extract the files:
cat 20bn-something-something-v1-?? | tar zx

**Comment by Fazlik995** [comment]
@swathikirans did you try training Diving48 dataset using 24 frames? If so, the results did not increase?

I wonder why you choose 16 frames, not 24 frames?

**Comment by swathikirans** [comment]
We did not try using 24 frames.

**Comment by Fazlik995** [comment]
@swathikirans I got the same results as your paper using epoch=30.

Thank you for sharing such great work!
